examples/experimental/CrypTen/CrypTen - Traing Encrypted Neural Network - Plans.ipynb

```diff
@@ -0,0 +1,543 @@
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[%] Connecting to workers ...\n",
+      "[+] Connected to workers\n",
+      "[%] Sending labels and training data ...\n",
+      "[+] Data ready\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Syft workers\n",
+    "print(\"[%] Connecting to workers ...\")\n",
+    "ALICE = NodeClient(hook, \"ws://localhost:3000\")\n",
+    "BOB = NodeClient(hook, \"ws://localhost:3001\")\n",
+    "print(\"[+] Connected to workers\")\n",
+    "\n",
+    "print(\"[%] Sending labels and training data ...\")\n",
+    "\n",
+    "# Prepare the labels\n",
+    "label_eye = torch.eye(2)\n",
+    "labels = torch.load(\"/tmp/train_labels.pth\")\n",
+    "labels = labels.long()\n",
+    "labels_one_hot = label_eye[labels]\n",
+    "\n",
+    "# Prepare and send training data\n",
+    "alice_train = torch.load(\"/tmp/alice_train.pth\").tag(\"alice_train\")\n",
+    "alice_ptr = alice_train.send(ALICE)\n",
+    "bob_train = torch.load(\"/tmp/bob_train.pth\").tag(\"bob_train\")\n",
+    "bob_ptr = bob_train.send(BOB)\n",
+    "\n",
+    "print(\"[+] Data ready\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check the data shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One entry from the MNIST dataset contains 28x28 features. Those are splitted accross our workers.\n",
+    "\n",
+    "We can check it out by running the next cell!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Alice data shape torch.Size([100, 28, 20])\n",
+      "Bob data shape torch.Size([100, 28, 8])\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"Alice data shape {alice_train.shape}\")\n",
+    "print(f\"Bob data shape {bob_train.shape}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initialize a dummy model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instanciate a model and create a dummy input that could be forwarded through it. This is needed to build the CrypTen model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dummy_input = torch.empty(1, 1, 28, 28)\n",
+    "pytorch_model = ExampleNet()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the CrypTen computation\n",
+    "\n",
+    "We need to specify for the ```run_multiworkers``` decorater:\n",
+    "* the workers that will take part in the computation\n",
+    "* the master address, this will be used for their syincronization\n",
+    "* the insntiated model that will be sent\n",
+    "* a dummy input for the model\n",
```

Comment:
```suggestion
    "We need to specify for the ```run_multiworkers``` decorator:\n",
    "* the workers that will take part in the computation\n",
    "* the master address, this will be used for their synchronization\n",
    "* the instantiated model that will be sent\n",
    "* a dummy input for the model\n",
```